src/coreclr/jit/gentree.cpp

```diff
@@ -25150,6 +25175,40 @@ bool GenTreeHWIntrinsic::OperIsMemoryStoreOrBarrier() const
     }
 }
 
+//------------------------------------------------------------------------
+// OperIsBroadcastScalar: Is this HWIntrinsic a broadcast node from scalar.
+//
+// Return Value:
+//    Whether "this" is a broadcast node from scalar.
+//
+bool GenTreeHWIntrinsic::OperIsBroadcastScalar() const
+{
+#if defined(TARGET_XARCH)
+    NamedIntrinsic intrinsicId = GetHWIntrinsicId();
+    switch (intrinsicId)
+    {
+        case NI_AVX2_BroadcastScalarToVector128:
+        case NI_AVX2_BroadcastScalarToVector256:
```

Comment:
This is missing some of the new broadcast scenarios that were covered.

Reply:
removed this method since it's never been called.